README.md

```diff
@@ -1 +1,155 @@
-# cryptollar
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+<img src="./src/assets/cryptollar.png" alt="cryptollar logo" width="140"  height="auto" />
+  <br/>
+  <h1><b>CRYPTOLLAR</b></h1>
+</div>
+
+</div>
+# 📗 Table of Contents
```

Comment:
- [ ] [OPTIONAL]
Kindly improve upon the look of the underlined section so as the make the ReadMe appear more professional.
  
![image](https://user-images.githubusercontent.com/12745474/234256900-25f86c14-ec41-4273-acc3-08a97096dbd2.png)